src/Lesson_8/Arrays_2/ArraysPartTwo.java

```diff
@@ -1,4 +1,76 @@
 package Lesson_8.Arrays_2;
 
 public class ArraysPartTwo {
+
+    public static int[] arr = generateRandomArray();
+    public static char[] reverseFullName = { 'n', 'a', 'v', 'I', ' ', 'v', 'o', 'n', 'a', 'v', 'I'};
+
+    public static void textSolvedTask(int task) {
+        System.out.print("\nРешение задачи #" + task + ": ");
+    }
+
+    public static int[] generateRandomArray() {
+        java.util.Random random = new java.util.Random();
+        int[] arr = new int[30];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+
+    public static void task1() {
+        int payments = 0;
+
+        textSolvedTask(1);
+        for (int payment : arr) {
+            payments += payment;
+        }
+        System.out.printf("Сумма трат за месяц составила %d рублей", payments);
+    }
+
+    public static void task2() {
+        int min = 200_000;
+        int max = -1;
+
+        textSolvedTask(2);
+
+        for (int payment : arr) {
+            if (payment > max) {
+                max = payment;
+            }
+
+            if (payment < min) {
+                min = payment;
+            }
+        }
+        System.out.printf("Минимальная сумма трат за день составила %d рублей. " +
+                "Максимальная сумма трат за день составила %d рублей", min, max);
+    }
+
+    public static void task3() {
+        float payments = 0.0f;
+        int daysInMonth = 30;
```

Comment:
откуда это число? не нужно брать с потолка, дан массив, значит надо просто взять его длину
```suggestion
        int daysInMonth = arr.length;
```

Reply:
Число взято из задания, где усреднено количество месяцев до 30. Но так с тобой соглашусь, да. Иначе вероятность ошибки возникает.